scripts/lib.star

```diff
@@ -272,6 +272,23 @@ def build_storybook_step(edition, ver_mode):
         ],
     }
 
+def verify_storybook_step(edition, ver_mode):
```

Comment:
Could you include the command `./bin/grabpl verify-storybook' in the build-storybook and publish-storybook steps instead? I don't see any benefit to making its own step. Every step introduces some risk/complexity/performance degradation.